### PR TITLE
Adding new packages to base package.json

### PR DESCRIPTION
**Pull request by amywalkerdev**
**Overall change:** Adds some newer Psammead packages to the base package.json

**Code changes:**

- @bbc/psammead-heading-index and @bbc/psammead-social-embed have been added to the base package.json

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,13 +76,15 @@
     "@bbc/psammead-media-player": "^2.7.11",
     "@bbc/psammead-most-read": "^4.1.4",
     "@bbc/psammead-navigation": "^6.0.11",
+    "@bbc/psammead-oembed": "0.1.0-alpha.1",
```
Comment (amywalkerdev) [inline package.json]: @simonsinclair Do we still need the psammead oembed component? I know it's not being used anywhere at the moment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.170 | [PR#](https://github.com/bbc/psammead/pull/) Adding new packages to the base package.json |
```
Comment (amywalkerdev) [inline CHANGELOG.md]:
```suggestion
| 2.0.170 | [PR#3571](https://github.com/bbc/psammead/pull/3571) Adding new packages to the base package.json |
```

**Comment by thekp** [comment]
This PR has reminded how outdated this list of packages is: https://github.com/bbc/psammead/blob/latest/packages/README.md#list-of-all-packages

File: package.json (hunk 1 above)
Comment (simonsinclair) [reply package.json]: It's not needed. 🙂  
I discussed removing it with @jamesdonoh earlier in the year, after we went down another route with Social Embeds. We decided it wouldn't hurt to leave it, but I wouldn't be against it being removed – especially as it never came out of alpha and wouldn't be hard to build again.

Comment (amywalkerdev) [reply package.json]: Cool, I'll remove it from here for now, you can remove the component itself later if you need to :)

**Review by joshcoventry — APPROVED** [review]

**Review by joshcoventry — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

**Review by OlgaLyubin — APPROVED** [review]
LGTM

**Pull request merged**